tempesta_fw/Makefile

```diff
@@ -16,7 +16,7 @@
 # this program; if not, write to the Free Software Foundation, Inc., 59
 # Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 
-EXTRA_CFLAGS += -Werror -I$(src)/../sync_socket -I$(src)/../tempesta_db
+EXTRA_CFLAGS +=  -I$(src)/../sync_socket -I$(src)/../tempesta_db
```

Comment:
I just want to comment out a single function call. However with ```-Werror``` the code won't compile as I get a compiler warning ```tempesta/tempesta_fw/http.c:690:1: warning: 'tfw_http_add_or_append_forwarded_for' defined but not used [-Wunused-function]```.